Gruntfile.js

```diff
@@ -75,7 +75,7 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('build_dev', ['webpack:dev', 'ts:server', 'less', 'concat:js_dependencies']);
-  grunt.registerTask('build_min', [/*'webpack:prod', */'ts:server', 'less', 'concat:js_dependencies_min']);
+  grunt.registerTask('build_min', ['webpack:prod', 'ts:server', 'less', 'concat:js_dependencies_min']);
```

Comment:
You can remove the `webpack` task from the `npm postinstall` script in `package.json`